cmd/helpers/output/output.go

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/*
+	output.Trace("Something very low level.")
+	output.Debug("Useful debugging information.")
+	output.Info("Something noteworthy happened!")
+	output.Warn("You should probably take a look at this.")
+	output.Error("Something failed but I'm not quitting.")
+	// Calls os.Exit(1) after logging
+	output.Fatal("Bye.")
+	// Calls panic() after logging
+	output.Panic("I'm bailing.")
+*/
+
+package output
```

Comment:
>output [](http://example.com/codeflow?start=8&length=6)

do you expect the implementation packages might need to use this same output framework? If so it probably belongs in /internal or /pkg

This is a good time to establish rules for code layout to align with application logic and layering.

cmd is on top, pkg under that, internal under that. No layer can reference a layer above it.

Reply:
we could also have a contract with stuff under /pkg to accept io.Writer for output and errors and have the /cmd implementations pass the appropriate implementation down.